package.json

```diff
@@ -2,7 +2,7 @@
     "name": "somename",
     "version": "1.0.0",
     "dependencies": {
-         "amdefine": "0.1.0",
```

Comment:
I might have actually left these as they were since they were since they were just used for tests. IIRC, aerospike was chosen to test a native code integration. I'm not really familiar with that library but hopefully upgrading it doesn't change whether it uses native code or not so that we're still actually testing that functionality